elasticache/cdk.json

```diff
@@ -0,0 +1,4 @@
+{
+  "app": "./elasticache",
```

Comment:
I suppose it doesn't matter, but would memcache also get this name?

Reply:
As we discussed yesterday the idea is for each elasticache service (redis/memcache) to share a common binary name to simplify the cdk-runner process.